openwisp_utils/ci.py

```diff
@@ -0,0 +1,34 @@
+"""
+This file contains functions that are required by
+multiple openwisp repositories for testing
+during continuous integration
+"""
+import os
+import re
+
+
+def CheckMigrationName():
+    """
+    Check if the migration files created have a descriptive
+    name; if default name pattern is found, exit with 1
+    """
+    MIGRATION_PATH = os.environ["MIGRATION_PATH"]
+    PROJECT_NAME = os.environ["TRAVIS_REPO_SLUG"]
+
+    # Get INGORE_SET
+    if "openwisp-users" in PROJECT_NAME:
+        IGNORE_SET = set(["0002_auto_20180508_2017.py"])
```

Comment:
I don't remember how many migrations contain `auto` in the openwisp2 modules, this would become hard to maintain.
I think we can do something simpler which will still help us out: check only the last `<n>` files (in alphabetical order) and raise an issue only if one of that file contains. The `<n>` can be passed to the script as an argument and would have the default of `1`. What do you think?